src/extension/src/constants.ts

```diff
@@ -268,6 +268,8 @@ export const CONSTANTS = {
   VALIDATION_LIMIT: 3
 };
 
+export const APP_NAME_VALIDATION_LIMIT = 20;
```

Comment:
would it is ok if you change this constant to `PROJECT_NAME_VALIDATION_LIMIT` cause app name sounds like azure related stuff lol

Reply:
Sure. lol